api/v1/Makefile.protoc

```diff
@@ -41,4 +41,7 @@ all:
 			--go-grpc_out=require_unimplemented_servers=false,paths=source_relative:. \
 			--go-json_out=orig_name=true,paths=source_relative:. \
 			$${proto}; \
+		$(PROTOC) --plugin=$(GOPATH)/bin/protoc-gen-gotag -I $(HUBBLE_PROTO_PATH) \
```

Comment:
Can this plugin be included in $HUBBLE_PROTOC_PLUGINS and run as part of the previous command?

Reply:
I tried that, but it fails then with an error of multiple attempts to write to the same file. 
I believe it is mentioned on the plugin page as a known limitation https://github.com/srikrsna/protoc-gen-gotag#note